packages/playground/src/pages/define-page/utils.ts

```
@@ -0,0 +1,4 @@
+export interface HelloWorld {
+  hello: string;
```

ESLint

[error] Unexpected separator (;).

(style/member-delimiter-style)